src/main/java/seedu/address/model/assignment/DueDate.java

```diff
@@ -27,7 +28,8 @@
 
     protected static final DateTimeFormatter PARSE_DATE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy");
     protected static final DateTimeFormatter PARSE_TIME_FORMAT = DateTimeFormatter.ofPattern("HHmm");
-    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a");
+    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a",
+            Locale.ENGLISH);
```

Comment:
Not sure about the "Locale.ENGLISH" part, but is this part supposed to be included for us as well?